src/components/loginRedirect/LoginRedirect.jsx

```diff
@@ -69,14 +68,14 @@ export const LoginRedirect = ({
         m: mobile,
         nonce: Date.now(),
       }
-      responseurlDetails.sig = web3Instance.eth.accounts.sign(responseurlDetails, goodWallet.accounts[0].privateKey)
+      responseurlDetails.sig = goodWallet.sign(responseurlDetails, goodWallet.accounts[0].privateKey)
       delete responseurlDetails.sig.message
     } else {
       responseurlDetails = {
         error: 'Authorization Denied',
       }
     }
-    if (Platform.OS === 'ios' || Platform.OS === 'android') {
+    if (isMobileNative()) {
```

Comment:
i dont think isMobileNative is a function

Reply:
Yeah fixed